Dockerfile

```diff
@@ -31,13 +31,14 @@ COPY config/ config/
 COPY pkg/ pkg/
 COPY cmd/ cmd/
 COPY third_party/ third_party/
+COPY .git/ .git/
 
-RUN mkdir bin; CGO_ENABLED=0 go build -o bin ./cmd/...
+RUN apt-get update && apt-get install -y jq && mkdir bin && CGO_ENABLED=0 make
```

Comment:
I'd put the `make` invocation as its own `RUN` line so the `apt` & `mkdir` calls can be cached separately. WDYT?

Reply:
Yes, makes sense. Updated.